### ISSUE TITLE
Keyboard shortcuts: should be a table so NVDA reads all the properties properly

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#52154]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Screenreader,NVDA[2018.2],JAWS[18.0.5038],MAS1.3.1

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to Settings icon in Activity Bar->Press Enter->Select "Keyboard Shortcuts" from dropdown.
3.Using keyboard key such as "tab" navigate to "Keyboard shortcuts" table data.
4.Observed that Nvda is not reading word "Column" to differentiate the cloumn name and column data.

**Actual:**
While reading through the screen reader such as NVDA, it reads the column name along with the cloumn data withought differentiating the word "Cloumn".
here, NVDA is reading "Command is Add Cursor Above. Keybinding is R. Source is User. When is editorTextFocus.  level 1" in this "Command" is the cloumn name and "Add Cursor Above. Keybinding is R. Source is User. When is editorTextFocus.  level 1" is the cell value.

Same issue is observed in jaws

**Expected:**
NVDA should reads the column name and cell value along with the "Column" word.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
User will not able to differentiate the table cells value and table header name.

**MAS Reference:**
MAS1.3.1:  [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_KeyboardShortcuts_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2110059/Accessibility.A11y_VSCode_KeyboardShortcuts_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Leaving this to @sandy081 to decide if it is low hanging fruit

**Comment by sandy081** [comment]
All the fields are being read.. See the screenshot

![image](https://user-images.githubusercontent.com/10746682/42382387-6bc35ba8-8134-11e8-951e-f9adf524019e.png)

Not sure what more is needed.

@AccessibilityTestingTeam-TCS FYI

**Comment by AccessibilityTestingTeam-TCS** [comment]
Here the issue is while reading through the screenreader, nvda is reading only the cells value continue with the header name but not differentiating it with word "column" and because of this user will not understand what is the header name and what is the column name.For more information please check the attached video also.

**Comment by sandy081** [comment]
@AccessibilityTestingTeam-TCS But what is read is properly understandable as it clearly says that `<Column Name> is <value>`. May I know what value does it add if it says `<Column Name` column is <value>` ?

**Comment by isidorn** [comment]
Let's reopen and give more time to @AccessibilityTestingTeam-TCS to answer

**Comment by isidorn** [comment]
@AccessibilityTestingTeam-TCS can you please answer @sandy081 question

**Comment by AccessibilityTestingTeam-TCS** [comment]
Here the issue is while reading through the screenreader, nvda is reading only the cells value continue with the header name but not differentiating it with word "column" and because of this user will not understand what is the header name and what is the column name.For more information please check the attached video also.
for eg. screenreader reading it as "command is Add source Above,keybinding is R...etc" not differentiating the column name and cells value.it should read "command header name" also please check the attachment in issue

**Comment by sandy081** [comment]
@AccessibilityTestingTeam-TCS Are you saying that when it is read as `Command is Add Source Above, Keybinding is control shift j, Source is Default, When is ....` is not understandable? These columns are not tab-able. So not sure the need of mentioning the term column.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Sure, you can please check it because it look like table only and if is is table then it should read in table format

**Comment by sandy081** [comment]
I think for an accessibility user it should be good what the NVDA says right?

**Comment by AccessibilityTestingTeam-TCS** [comment]
Right...but it's look like table and reading format is also like table so it should read the word column while reading the columns name along with cell values

**Comment by sandy081** [comment]
Discussed with @isidorn and the current narration makes sense and adding `column` makes it noisy. Hence closing this.
